### PR TITLE
Feat : Edit, Delete and Background Color Change.

### PR DESCRIPTION
**Pull request by anuj-kumary**
Feature Added:-
   - Edit Note
   - Delete Note
   - Background Chnage.

ScreenShot:-
Screenshot from 2022-03-26 15-17-28

Screenshot from 2022-03-26 15-17-39

### CODE REVIEW

File: src/components/DisplayNote/NoteCard.css [hunk 1]
```diff
@@ -7,4 +7,9 @@
 .card {
     height: min-content;
     margin: 1rem 0;
+}
+
+.btn {
+    padding: .6rem .9rem;
+    margin: 0.6rem 0.5rem 0.2rem 1rem;
```
Comment (varnita619) [inline src/components/DisplayNote/NoteCard.css:14]: Liked Consistency of units

File: src/components/Main/Main.jsx [hunk 2]
```diff
@@ -13,10 +13,12 @@ export const Main = () => {
   const { dispatch } = useData();
   const date = new Date();
   const navigate = useNavigate();
+
```
Comment (varnita619) [inline src/components/Main/Main.jsx:16]: Good use of navigate

**Review by varnita619 — APPROVED** [review]
Liked the folder structure
Checks are correctly implemented

**Review by Shah-Arsalan — APPROVED** [review]
Code seems neat . Nothing that I can suggest to change . You can go ahead and merge the code

File: src/components/Main/Main.jsx [hunk 3]
```diff
@@ -34,17 +36,33 @@ export const Main = () => {
...
-      note: note,
-      encodedToken: token,
-    });
-    console.log(response);
-    if (response.status === 201) {
-      dispatch({
-        type: 'ADD_NOTE',
-        payload: { noteList: response.data.notes },
+    if (!note._id) {
+      console.log(note._id);
```
Comment (Rohanmond) [inline src/components/Main/Main.jsx:40]: remove console

**Review by Rohanmond — APPROVED** [review]

File: src/components/Main/Main.jsx (hunk 3 above)
Comment (anuj-kumary) [reply src/components/Main/Main.jsx:40]: Done

**Pull request merged**